tmt/export.py

```diff
@@ -513,15 +604,173 @@ def export_to_nitrate(test):
+            verifies_bug_ids, int(nitrate_case.id), NITRATE_TRACKER_ID)
+
+
+def export_to_polarion(test):
+    """ Export fmf metadata to a Polarion test case """
+    import_polarion()
+
+    # Check command line options
+    create = test.opt('create')
+    link_bugzilla = test.opt('bugzilla')
+    project_id = test.opt('project_id')
+    dry_mode = test.opt('dry')
+    duplicate = test.opt('duplicate')
+
+    polarion_case = None
+    if not duplicate:
+        polarion_case = get_polarion_case(test.node, project_id)
+    summary = prepare_extra_summary(test)
+
+    if not polarion_case:
+        if create:
+            if not project_id:
+                raise ConvertError(
+                    "Please provide project_id so TMT knows which "
+                    "Polarion project to use for this test case")
+            if not dry_mode:
+                polarion_case = create_polarion_case(
+                    summary, project_id=project_id)
+            else:
+                echo(style(
+                    f"Test case '{summary}' created.", fg='blue'))
+            test._metadata['extra-summary'] = summary
+        else:
+            raise ConvertError(
+                f"Polarion test case id not found for {test} "
+                f"(You can use --create option to enforce creating testcases)")
+
+    # Title
+    if not dry_mode and polarion_case.title != test.summary:
+        polarion_case.title = test.summary
+    echo(style('title: ', fg='green') + test.summary)
+
+    # Description
+    description = summary
+    if test.description:
+        description += ' - ' + test.description
+    if not dry_mode:
+        polarion_case.description = description
+    echo(style('description: ', fg='green') + description)
+
+    # Automation
+    if test.node.get('extra-task'):
+        automation_script = test.node.get('extra-task')
+        automation_script += ' ' + test.fmf_id['url']
+    else:
+        automation_script = test.fmf_id['url']
+    if not dry_mode:
+        polarion_case.caseautomation = 'automated'
+        polarion_case.automation_script = automation_script
+    echo(style('script: ', fg='green') + automation_script)
+
+    # Components
+    if not dry_mode:
+        polarion_case.caselevel = 'component'
+        polarion_case.testtype = 'functional'
+        if test.component:
+            polarion_case.casecomponent = [test.component[0]]
+    echo(style('components: ', fg='green') + ' '.join(test.component))
+
+    # Tags and Importance
+    if not dry_mode:
+        if test.tier is not None:
+            if int(test.tier) <= 1:
+                polarion_case.caseimportance = 'high'
+            elif int(test.tier) == 2:
+                polarion_case.caseimportance = 'medium'
+            else:
+                polarion_case.caseimportance = 'low'
+            test.tag.append(f"Tier{test.tier}")
+        else:
+            polarion_case.caseimportance = 'medium'
+
+    test.tag.append('fmf-export')
```

Comment:
I guess this should be `polarion-export`? If the tag is needed...

Reply:
it isn't needed, just mirrored nitrate export functionality